packages/data-context/test/unit/code-generator.spec.ts

```diff
@@ -244,7 +244,7 @@ describe('code-generator', () => {
     }).not.throw()
   })
 
-  it('should generate from scaffoldIntegration', async () => {
+  xit('should generate from scaffoldIntegration', async () => {
```

Comment:
Should this be disabled? Can we put a comment here if so?

Reply:
Fixed it up, weird place since cypress-kitchensink is not using `e2e`, but still `integration`, or at least the one on npm.

Reply:
https://github.com/cypress-io/cypress/pull/20134/commits/ae16f30a8ce6acff66e26311673b9fa432116bbf